.github/workflows/test.yml

```
@@ -0,0 +1,30 @@
+name: Test
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.15.0
```

What do you think if using the [node-version-file](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file) instead a fixed version instead?

```suggestion
          node-version-file: '.nvmrc'
```

---

love that, thanks @v1v